panoramapublic/src/org/labkey/panoramapublic/pipeline/CopyExperimentFinalTask.java

```diff
@@ -544,7 +544,7 @@ private void updatePermissions(User formSubmitter, ExperimentAnnotations targetE
         assignPanoramaPublicSubmitterRole(newPolicy, log, targetExperiment.getSubmitterUser(), targetExperiment.getLabHeadUser(),
                 formSubmitter); // User that submitted the form. Can be different from the user selected as the data submitter
 
-        SecurityPolicyManager.savePolicy(newPolicy);
+        SecurityPolicyManager.savePolicy(newPolicy, pipelineJobUser);
```

Comment:
It should be OK to pass in the User that started the pipeline job instead of User.getAdminServiceUser().  I will revert the changes in my commit if tests fail due to this change.

Reply:
Thanks!